saml2/saml2-service-provider/src/test/java/org/springframework/security/saml2/provider/service/web/authentication/OpenSamlAuthenticationRequestResolverTests.java

```diff
@@ -65,6 +65,7 @@ public void resolveAuthenticationRequestWhenSignedRedirectThenSignsAndRedirects(
 		assertThat(result.getSigAlg()).isEqualTo(SignatureConstants.ALGO_ID_SIGNATURE_RSA_SHA256);
 		assertThat(result.getSignature()).isNotEmpty();
 		assertThat(result.getBinding()).isEqualTo(Saml2MessageBinding.REDIRECT);
+		assertThat(result.getID()).isNotEmpty();
```

Comment:
These tests don't appear to be running. Intellij couldn't run them and when I added an exception to be thrown in the tests, there are no failures running with gradle. Is that something that should be looked into for this ticket or a new issue created?

Reply:
You were correct. I updated to use JUnit 5 and the tests are running now. There were two failing tests when I ran them with JUnit 5. ```resolveAuthenticationRequestWhenSignedThenCredentialIsRequired``` and ```resolveAuthenticationRequestWhenSHA1SignRequestThenSigns``` because ```OpenSamlAuthenticationRequestResolver#resolve``` does not check if authnRequestConsumer is null before calling it on line 132. I added an empty consumer to fix this issue.